Bottom: e2561f0d28182c6c5fe20494c6c7343ffc058fbf
Top:    31c37d90f820fb92df3f201d43ada7c51089352d
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-05 22:30:54 +0000

Refresh of extend-object3d-code-to

---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index c6fb480..1d1eeaf 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -259,10 +259,11 @@ get_contour_edge_n_is_round (PLINE *contour, int n)
     }
 #endif
 
-  while (n > 0) {
-    edge = edge->next; /* The VNODE structure is circularly linked, so wrapping is OK */
-    n--;
-  }
+  while (n > 0)
+    {
+      edge = edge->next; /* The VNODE structure is circularly linked, so wrapping is OK */
+      n--;
+    }
 
   return edge->is_round;
 }
@@ -308,10 +309,11 @@ get_contour_edge_n_round_geometry_in_step_mm (PLINE *contour, int n, double *cx,
     }
 #endif
 
-  while (n > 0) {
-    edge = edge->next; /* The VNODE structure is circularly linked, so wrapping is OK */
-    n--;
-  }
+  while (n > 0)
+    {
+      edge = edge->next; /* The VNODE structure is circularly linked, so wrapping is OK */
+      n--;
+    }
 
   center[0] = edge->cx;
   center[1] = edge->cy;
